FBTweak/_FBTweakCollectionViewController.m

```diff
@@ -74,12 +77,48 @@ - (void)viewWillDisappear:(BOOL)animated
   [_keyboardManager disable];
 }
 
+- (BOOL)_supportsUpdates {
+  return [self.tweakCategory respondsToSelector:@selector(updateWithCompletion:)];
+}
+
+- (void)_updateCategory {
+  self.refreshControl.attributedTitle =
+      [[NSAttributedString alloc] initWithString:@"Refreshing..."];
+  [self.refreshControl beginRefreshing];
+  [self.tweakCategory updateWithCompletion:^(NSError * _Nullable error) {
+      dispatch_async(dispatch_get_main_queue(), ^{
+        [self.refreshControl endRefreshing];
+        if (error) {
+          [self presentViewController:[self errorAlertControllerWithError:error] animated:YES
+                           completion:nil];
+        }
+      });
+    }];
+}
+
+- (UIAlertController *)errorAlertControllerWithError:(NSError * _Nonnull)error {
```

Comment:
Remove the `_Nonnull`.

Reply:
mmmm in `.m` files apparently I can use nullability annotations, I think we should keep them.